daprdocs/content/en/reference/components-reference/supported-bindings/alicloud-nacos.md

```diff
@@ -0,0 +1,115 @@
+---
+type: docs
+title: "Alibaba Cloud Nacos binding spec"
+linkTitle: "Alibaba Cloud Nacos"
+description: "Detailed documentation on the Alibaba Cloud Nacos binding component"
+aliases:
+- "/operations/components/setup-bindings/supported-bindings/alicloud-nacos/"
```

Comment:
```suggestion
```

You can delete this as it's the first time this page is getting created and there wasn't a prior location.